test/e2e/storage/csi_mock_volume.go

```diff
@@ -193,7 +193,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			driverAttachable bool
 			deployDriverCRD  bool
 		}{
-			{
```

Comment:
I don't think these need to be commented out.  They currently have a [Feature] tag on them so are not run by default, and I have removed the Feature tag whitelisting on the alpha suites.

Once update the sidecars, then all we need to do is remove the [Feature] tag from the test suite like I have [here](https://github.com/msau42/kubernetes/commit/3057a348ed9364658b030973454b43250bf37090)

Reply:
Ok.  Will uncomment them out.